packages/article-skeleton/src/native-ad.web.js

```diff
@@ -0,0 +1,36 @@
+const insertNativeAd = (children) => {
+	const child = children.find(item => item.name === 'paywall');
+	
+  if (!child) {
+    return children;
+  }
+
+	const clonedChildren = [...children];
```

Comment:
I would move this to line 2 and then do 
```suggestion
	const child = clonedChildren.find(item => item.name === 'paywall');
	if (!child) {
    return clonedChildren;
  }
```

Reply:
has been updated